storage_controller/src/service/safekeeper_service.rs

```diff
@@ -146,10 +146,16 @@ impl Service {
             "Got {} non-successful responses from initial creation request of total {total_result_count} responses",
             remaining.len()
         );
-        if remaining.len() >= 2 {
+        let quorum_size = match timeline_persistence.sk_set.len() {
+            1 => 1,
+            2 => 2,
+            _ => (timeline_persistence.sk_set.len() * 3) / 2,
```

Comment:
Let's say we picked three safekeepers in `safekeepers_for_new_timeline`.
In that case `quorum_size` would be `int(3 * 3 / 2) = 4`. That can never be reached.

What am I missing?

Reply:
I mixed up the two, it should be * 2 / 3, basically to get 66% of safekeepers on board.

Reply:
Why did the new test pass previously? I guess it is not checking if quorum was achieved. Should it?

Reply:
i think it worked because of the wrong quorum check which has recently been corrected.